src/streams/send.rs

```diff
@@ -382,8 +389,23 @@ impl StreamTx {
         // TODO: Can we remove this?
         let header_ref = &mut next.pkt.header;
 
+        // <https://webrtc.googlesource.com/src/+/refs/heads/main/modules/rtp_rtcp/source/rtp_sender.cc#537>
+        // BUNDLE requires that the receiver "bind" the received SSRC to the values
+        // in the MID and/or (R)RID header extensions if present. Therefore, the
+        // sender can reduce overhead by omitting these header extensions once it
+        // knows that the receiver has "bound" the SSRC.
+        // <snip>
+        // The algorithm here is fairly simple: Always attach a MID and/or RID (if
+        // configured) to the outgoing packets until an RTCP receiver report comes
+        // back for this SSRC. That feedback indicates the receiver must have
+        // received a packet with the SSRC and header extension(s), so the sender
+        // then stops attaching the MID and RID.
+
         // This is true also for RTX.
-        header_ref.ext_vals.mid = Some(mid);
+        if !recv_acked_ssrc {
```

Comment:
Do we want to pull this from the stats with a slightly more opaque check? `if self.stats.rtt.is_some()` ? We can also pull this commit into its own PR

Reply:
This is good. I'm renaming it to `remote_acked_ssrc`